src/utils/parse5.js

```diff
@@ -49,3 +73,42 @@ export function createTextNode (content, parent) {
         parentNode: parent
     };
 }
+
+export function removeAttr (el, attr) {
+    for (var i = 0; i < el.attrs.length; i++) {
+        if (getAttrName(el.attrs[i]) === attr) {
+            el.attrs.splice(i, 1);
+
+            return;
+        }
+    }
+}
+
+export function getAttr (el, attr) {
```

Comment:
`attr` arg should be renamed to `name`. And we have code duplication that searches by attr name. Maybe we will have internal `findAttr` method that will return `attr` object and use it `getAttr` and `setAttr` methods?